app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -71,3 +73,49 @@ export const fetchPlanAction = (url, id) => {
 export const resetPlanStateAction = () => ({
   type: RESET_PLAN_STATE
 });
+
+export const downloadLogAction = (task, addNotificationAction) => dispatch =>
```

Comment:
An alternative to way to get access to `addNotificationAction` could be to directly import the action into this file from `NotificationListActions.js.

That would eliminate the need to pass the action down through the component tree as a prop

Reply:
@michaelkro Sure, will give that a try. Thanks.

Reply:
@michaelkro So directly importing `addNotificationAction` and calling it directly did not work. So I am using `dispatch` on `V2V_NOTIFICATION_ADD ` instead.
Is that OK?

Reply:
Absolutely!  In fact, I think this makes much more sense than my suggestion (sorry about that by the way 😬, I should have double checked before commenting)

Reply:
No problem! Thanks for confirming!!